src/main/java/com/natalia/proyectoSpringBoot/services/IUserService.java

```diff
@@ -0,0 +1,23 @@
+package com.natalia.proyectoSpringBoot.services;
+
+import com.natalia.proyectoSpringBoot.dto.UserDTO;
+import com.natalia.proyectoSpringBoot.models.User;
+
+import java.util.List;
+import java.util.Map;
+
+public interface IUserService {
+    List<User> getUsers();
+
+    List <UserDTO> getUsersInfo();

```

Comment:
Estaria bueno si puedieras agregar algun comentario que explique cual es la diferencia entre getUsers and getUserInfo